extensions/sql-database-projects/src/tools/buildHelper.ts

```diff
@@ -178,8 +178,7 @@ export class BuildHelper {
 		return this.extensionBuildDir;
 	}
 
-	public constructBuildArguments(projectPath: string, buildDirPath: string, sqlProjStyle: ProjectType): string {
-		projectPath = utils.getQuotedPath(projectPath);
+	public constructBuildArguments(buildDirPath: string, sqlProjStyle: ProjectType): string {
```

Comment:
why remove projectPath?

Reply:
project path is not using from the build arguments, It is being added in the project Controller before executing the task.